tests/Makefile

```diff
@@ -17,7 +17,7 @@ testBuildDir := $(testBinDir)/build
 
 # Set build vars
 utestIncludeDir := $(vendorDir)/include/utest
-linkFlags += -l core -l utils
+linkFlags += -l core -l utils -lrender
```

Comment:
why is this necessary?

Reply:
It isn't. This is an artifact from when I was experimenting with having this in `render`

Reply:
Removed it.